timetracker/entries/migrations/0001_initial.py

```diff
@@ -1,25 +1,51 @@
 # -*- coding: utf-8 -*-
```

Comment:
Hi Markus, I think it was when I was trying to add an author to the models. I didn't want to allow the author to be null. From the d31dd00 commit. I tried to run a manual migration but got stuck trying figure out where it was failing the migration. It asked to provide options 1 or 2 to manually set or ... (I can't recall). I looked back at previous examples but couldn't work out how this was getting stuck as there was no data in the database to get an existing user.